Assets/Scripts/Interactables/Dialogue/NPCDialogue/NPCReactiveBark.cs

```diff
@@ -214,6 +214,21 @@ private void LoadBarks(MissionsEnum missionEnum)
                 lostBark = BarkEvent.BrutusOfficeLostReation;
                 reactiveNoiseBark = BarkEvent.BrutusOfficeNoiseReaction;
                 break;
+                //case MissionsEnum.:
+                //if (isBrutus)
+                //{
+                    //idleBark = BarkEvent.BrutusGardenIdleBark;
+                    //spottedBark = BarkEvent.BrutusGardenSpottedBark;
+                    //lostBark = BarkEvent.BrutusGardenLostBark;
+                    //reactiveNoiseBark = BarkEvent.BrutusGardenNoiseBark;
+                //}
+                //else
+                //{
+                    //idleBark = BarkEvent.GardenIdleBark;
+                    //spottedBark = BarkEvent.GardenSpottedBark;
+                    //lostBark = BarkEvent.GardenLostBark;
+                    //reactiveNoiseBark = BarkEvent.GardenNoiseBark;
+                //}
```

Comment:
Remove this if not needed anymore?

Reply:
oh this is set up for the next mission haha